File: server/DocumentsKM/Models/GeneralDataPoint.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DocumentsKM.Models
{
    // Пункт общих указаний
    public class GeneralDataPoint
    {
        [Key]
        public Int16 Id { get; set; }

        // Раздел
        [Required]
        [ForeignKey("SectionId")]
        public virtual GeneralDataSection Section { get; set; }

        // Текст
        [Required]
        public string Text { get; set; }

        // Номер
        [Required]
        public Int16 OrderNum { get; set; }
    }
}

File: client/src/model/GeneralDataPoint.ts
import GeneralDataSection from './GeneralDataSection'

interface IGeneralDataPoint {
	id: number
	section: GeneralDataSection
	text: string
	orderNum: number
	hasLineBreak: boolean
}

class GeneralDataPoint {
	id: number
	section: GeneralDataSection
	text: string
	orderNum: number
	hasLineBreak: boolean

	constructor(obj?: IGeneralDataPoint) {
		this.id = (obj && obj.id) || 0
		this.section = (obj && obj.section) || null
		this.text = (obj && obj.text) || ''
		this.orderNum = (obj && obj.orderNum) || 0
		this.hasLineBreak = (obj && obj.hasLineBreak) || false
	}
}

export default GeneralDataPoint
